### DISCUSSION TITLE
error pada saat running aplikasi vs code

### DISCUSSION THREAD

**Discussion by Baguspramudito** [teamalgoritma/community#758] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/141464903/4e9ad1c9-d31b-410e-8d00-98d069ec264f)

Dear team algoritma, mohon dibantu saya ada miss dimana ya, info error sudah saya input semua

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @Baguspramudito,

Silahkan dicoba untuk melakukam save file `app.py` terlebih dahulu, kemungkinan file tersebut belum ke save sehingga masih ada bagian rumpang code yang kosong.

Semoga jawaban saya membantu!
